client/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -474,17 +475,18 @@ private void startLastSuccessfulTunnel() {
     try {
       final String tunnelId = tunnel.getString(TUNNEL_ID_KEY);
```

Comment:
The three keys are a bit useless and requires more wiring. Can we have a single TunnelConfig with a format like:

- id: string
- name: string
- transport: TransportConfig (the current config, should become an opaque string)

?

It simplifies the code and we will need that container for VPN-level settings, like DNS.

Reply:
IIUC the proposal is  to change the TunnelConfig struct here: https://github.com/Jigsaw-Code/outline-apps/blob/6e78697c7bea2d6aa7be7dc5b1223bc12fc665fb/client/src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/TunnelConfig.aidl#L19

That's do-able but it's a bigger change. Can that be done incrementally in a different PR or does it have to be in this PR?

Reply:
Sounds good.